### PR TITLE
[iOS] :recycle: Use the defined Typography on each screen.

### LINKED ISSUE #1144: [iOS] Use the defined Typography on each screen.
## Summary

- Use the Typography implemented in task #889 on each screen.

### PR DESCRIPTION
**Pull request by Corvus400**
## Issue
- close #1144 

## Overview (Required)
- The previously defined Typography style is now set up like Android.

## Movie (Optional)
Before | After
:--: | :--:
 |

### CODE REVIEW

File: app-ios/Modules/Sources/Theme/TextModifiers.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    }
+}
+
+public struct LetterSpacingModifier: ViewModifier {
+    var spacing: CGFloat
+    public func body(content: Content) -> some View {
+        content.padding(.leading, spacing/2)
+            .padding(.trailing, spacing/2)
+    }
+}
```
Comment (Corvus400) [inline app-ios/Modules/Sources/Theme/TextModifiers.swift:17]: An extension function to handle TextStyle defined in Typography has been added this time.

File: app-ios/Modules/Sources/Theme/Typography.swift [hunk 2]
```diff
@@ -25,19 +25,19 @@ public struct TextStyle {
...
-    public static let titleSmall = TextStyle(font: SwiftUI.Font.custom("Montserrat-SemiBold", size: 14), lineHeight: 20, letterSpacing: 0.1)
-    public static let labelLarge = TextStyle(font: SwiftUI.Font.custom("Montserrat-SemiBold", size: 14), lineHeight: 20, letterSpacing: 0.1)
-    public static let labelMedium = TextStyle(font: SwiftUI.Font.custom("Montserrat-SemiBold", size: 12), lineHeight: 16, letterSpacing: 0.5)
-    public static let labelSmall = TextStyle(font: SwiftUI.Font.custom("Montserrat-SemiBold", size: 11), lineHeight: 16, letterSpacing: 0.5)
-    public static let bodyLarge = TextStyle(font: SwiftUI.Font.custom("Montserrat-Regular", size: 16), lineHeight: 24, letterSpacing: 0.15)
-    public static let bodyMedium = TextStyle(font: SwiftUI.Font.custom("Montserrat-Regular", size: 14), lineHeight: 20, letterSpacing: 0.25)
-    public static let bodySmall = TextStyle(font: SwiftUI.Font.custom("Montserrat-Regular", size: 12), lineHeight: 16, letterSpacing: 0.4)
+    public static let displayLarge = TextStyle(font: SwiftUI.Font.custom("Montserrat-Regular", size: 57), lineHeight: 64 - 57)
```
Comment (Corvus400) [inline app-ios/Modules/Sources/Theme/Typography.swift:28]: In the case of iOS, if the lineHeight value is used as it is, the display will be corrupted, so we have changed to use the value obtained by subtracting the lineHeight value from the Font size.

**Comment by Corvus400** [comment]
@ry-itto @tkhs0604 
For some reason, no reviewer has been assigned again, so please give us a code review. 🙇

File: app-ios/Modules/Sources/Theme/Typography.swift (hunk 2 above)
Comment (ry-itto) [inline app-ios/Modules/Sources/Theme/Typography.swift:28]: Sorry, I missed this review comment 🙏🏼 
Can you apply this for all vairables?

https://github.com/DroidKaigi/conference-app-2023/pull/1135#discussion_r1320616538

File: app-ios/Modules/Sources/Theme/TextModifiers.swift [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+            .lineSpacing(style.lineHeight)
+            .modifier(LetterSpacingModifier(spacing: style.letterSpacing ?? 0))
+    }
+}
+
+public struct LetterSpacingModifier: ViewModifier {
+    var spacing: CGFloat
+    public func body(content: Content) -> some View {
+        content.padding(.leading, spacing/2)
+            .padding(.trailing, spacing/2)
```
Comment (ry-itto) [inline app-ios/Modules/Sources/Theme/TextModifiers.swift:15]: `.horizontal` might be a better way. 👀  And maybe this modifier not needed if this line uses `.horizontal`.

```suggestion
        content.padding(.horizontal, spacing / 2)
```

File: app-ios/Modules/Sources/Theme/Typography.swift (hunk 2 above)
Comment (Corvus400) [reply app-ios/Modules/Sources/Theme/Typography.swift:28]: @ry-itto 
Sorry if I am wrong in my perception. :bow: 
I only commented on one area, but I have already corrected all the variables this time.

Comment (ry-itto) [reply app-ios/Modules/Sources/Theme/Typography.swift:28]: @Corvus400 
My review comment is bad.. (That review comment has two meanings..)
What I want to say is "using static access to font name might be better way". 🙏🏼 

Like this
`FontAssets.Montserrat.regular.font(size: 57`

Comment (Corvus400) [reply app-ios/Modules/Sources/Theme/Typography.swift]: @ry-itto 
Is [this](https://github.com/DroidKaigi/conference-app-2023/pull/1178/commits/e9e1c83a7d70e4509a67dcab4f2e5b2d16454d7b) correction correct?

**Review by ry-itto — APPROVED** [review]
LGTM! Thanks! 👍🏼

**Pull request merged**